epicure-frontend/src/components/IconsMeaning/IconsMeaning.scss

```diff
@@ -0,0 +1,65 @@
+@import '../../styles/mixins.scss';
+
+.icons-meaning-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background: #FAFAFA;
```

Comment:
set as variables

Reply:
I've created a variables file in the prior branches so once I will merge them I will apply the styling on all the components - is that ok?